packages/rancher-vsphere-csi/charts/templates/vsphere-csi-controller-rbac.yaml

```diff
@@ -36,7 +36,7 @@ rules:
     verbs: ["create", "get", "list", "watch", "update", "delete"]
   - apiGroups: ["apiextensions.k8s.io"]
     resources: ["customresourcedefinitions"]
-    verbs: ["get", "create"]
+    verbs: ["get", "create", "update"]
```

Comment:
Just want to clarify why this permission is needed; does the vsphere CSI controller automatically create CRDs, similar to what Rancher does?

Reply:
This is a change from upstream as well, so I'm not why this permission is needed (I don't see any CRDs in the chart). I tried to stick to what's upstream to make the chart easier to update.

Reply:
okay sounds good to me